contrib/readme.md

```diff
@@ -0,0 +1,75 @@
+grab bag directory for non core related platform specific non source code
+
+[Network Manager shims](NetworkManager)
+
+[apparmor profiles](apparmor)
+
+[default self signed network bootstrap files](bootstrap)
+
+[continuous integration scripts](ci)
+
+[cross compile cmake toolchains](cross)
+
+[embedded lokinet example code](liblokinet)
+
+[macos related hacks and associated hellscape items](macos)
+
+[static deps patch files](patches)
+
+[python scripts](py)
+
+[systemd-resolved profiles](systemd-resolved)
+
+android build shim:
+
+* android.sh
+* android-configure.sh
+
+windows via mingw cross compiler build shim:
+
+* windows.sh
+* windows-configure.sh
+
+macos build shim:
+
+* mac.sh
+
+iphone build shim:
+
+* ios.sh
+
+linux cross compile build shim:
+
+* cross.sh
+
+apply multiple patches script:
+
+* apply-patches.sh
+
+bencode pretty-print script:
+
+* bencode-dump.py
+
+deb.oxen.io apt repo pubkey:
+
+* deb.oxen.io.gpg
+
+clang-format / jsonnetfmt / swiftformat helper, will check or correct code style:
+
+* format.sh
+
+git hooks:
+
+* git-hook-pre-push.sh
+
+base16 to base32z converter
```

Comment:
maybe include a link to a base32z spec page?

Reply:
also apparently (if wikipedia is correct, which is of course always true) it is actually called `z-base-32`?